### PR TITLE
Task 5

### PR DESCRIPTION
**Pull request by asrubel**

### CODE REVIEW

File: README [hunk 1]
```diff
@@ -3,4 +3,4 @@ Python
 Programming Development
 Task 1
 Task 2 & Task 3
-Task 4
\ No newline at end of file
+Task 4 & Task 5 (finished)
```
Comment (asrubel) [inline README:6]: Could be better...

Comment (1newchoppa) [inline README:6]: norm

**Review by 1newchoppa — APPROVED** [review]

**Review by asrubel — COMMENTED** [review]
Negative

File: README [hunk 2]
```diff
@@ -3,4 +3,4 @@ Python
 Programming Development
 Task 1
 Task 2 & Task 3
```
Comment (MaxymOdanets) [inline README:5]: не воно,треба переробить)

File: README (hunk 1 above)
Comment (MaksimKosyanchuk) [inline README:6]: done

File: README [hunk 3]
```diff
@@ -3,4 +3,4 @@ Python
...
-Task 4
```
Comment (Dima-Chuychenko) [inline README:6]: ne norm

File: README (hunk 1 above)
Comment (Leviaweek) [inline README]: ne norm

File: README (hunk 3 above)
Comment (valen-yany) [inline README:6]: Delete "-" please

**Review by MaksimKosyanchuk — COMMENTED** [review]
bad code

**Review by Dima-Chuychenko — COMMENTED** [review]
ne norm

**Review by valen-yany — COMMENTED** [review]
Delete all

File: README (hunk 1 above)
Comment (Leviaweek) [inline README]: Переделать

**Review by valen-yany — APPROVED** [review]

**Review by MaksimKosyanchuk — APPROVED** [review]
done

**Review by Leviaweek — APPROVED** [review]

**Review by MaxymOdanets — APPROVED** [review]

**Review by Dima-Chuychenko — APPROVED** [review]

**Review by Dima5612 — APPROVED** [review]

**Review by Leviaweek — APPROVED** [review]

**Review by Malaxits — APPROVED** [review]

**Review by MaxymOdanets — APPROVED** [review]

**Pull request merged**